### ISSUE TITLE
Issues about SVDKL_Regression

### ISSUE THREAD

**Issue by perhapslove716** [cornellius-gp/gpytorch#625]
Hi,

gpytorch/examples/05_Scalable_GP_Regression_Multidimensional/SVDKL_Regression_SVGP_CUDA.ipynb

based on this example I want to implement DKL regression on my own dataset. I saw the data size pf the example are as follows:
train_x    :      torch.Size([347899, 3])
train_y    :      torch.Size([347899])

But If there is two channels in my dataset like this:
train_x    :      torch.Size([39124,2,1200])
train_y    :      torch.Size([39124,1])
test_x    :      torch.Size([1517,2,1200])
test_y    :      torch.Size([1517,1])

I used a convolutional NN to extract the features( num_features also 2). And then put it in the GP layer with the same setting as the example. But the test result were really bad although the training loss was decreasing.

Which parameters should be adjusted??

**Comment by perhapslove716** [comment]
@jacobrgardner Could you give me some help? thanks a lot!!

**Comment by perhapslove716** [comment]
I have also saw the suggestions in issue SGPR outputs constant value.

But even I enhanced the inducing points and used MaternKernel. The result was still bad..

**Comment by gpleiss** [comment]
There are many possible issues that could be affecting your outputs, and it's very difficult to debug without more info.

I would suggest trying to train your model without a GP at first. If you train a standard neural network, are you able to learn the desired function?

**Comment by perhapslove716** [comment]
> There are many possible issues that could be affecting your outputs, and it's very difficult to debug without more info.
> 
> I would suggest trying to train your model without a GP at first. If you train a standard neural network, are you able to learn the desired function?

Yes, I have already tried to use my model without a GP. The result can be acceptable but not very good. This is also the reason why I want to add a GP to improve the performance. But I am a quite new learner of GP.

Is there any possibility that the dimension of my dataset too large? or any tricks can make sense?
